partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,193 @@
+    type=click.Path(exists=True),
+    help="Specify the package path (YAML file or directory with 'partcad.yaml')",
+)
+@click.option(
+    "--format",
+    help="Set the log prefix format",
+    type=click.Choice(["time", "path", "level"]),
+    default=None,
+    show_envvar=True,
+)
+@click.pass_context
+def cli(ctx, verbose, quiet, no_ansi, package, format):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)